utilities/boostHistHelpers.py

```diff
@@ -129,12 +135,18 @@ def addSystAxis(h, size=1, offset=0):
     hnew[...] = np.stack((newvals, newvars), axis=-1)
     return hnew
 
-def clipNegativeVals(h, clipValue=0):
-    hnew = hist.Hist(*h.axes, storage=hist.storage.Weight())
-    vals = h.values(flow=True)
-    vals[vals<0] = clipValue
-    hnew[...] = np.stack((vals, h.variances(flow=True)), axis=-1)
-    return hnew
+def clipNegativeVals(h, clipValue=0, createNew=False):
+    if createNew:
```

Comment:
Can the createNew options minimally duplicate code? It's just a question of whether hfinal = h or hfinal = hnew in the end isn't it?